native/src/main/java/io/ballerina/stdlib/task/actions/TaskActions.java

```diff
@@ -50,6 +50,7 @@
     private static String value = "1000";
 
     public static Object configureThread(Environment env, long workerCount, long waitingTimeInMillis) {
+        TaskLogger.clearGlobalLogger();
```

Comment:
Do we need to clear the global logger in before every method?

Reply:
Yes, If we will use class, There is enough to call only in the init but these all are Util methods. So we can call any method without any restriction.  And we are creating a logger to log our message. If any log will be available, then we have a chance to get quartz logs.  So we need to use before every method.

Reply:
Can't we have a static block and call this function

Reply:
A static block runs only once when a class is loaded into memory. So If we will use a static block,  we can remove the existing log when calling the method the first time. But we are creating logs to log the message in our module. If any log will be available, then we get quartz logs. So we need to remove those logs also. Therefore we can't use static block.

Reply:
Okay